helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/tests/secret-elasticsearch_test.yaml

```diff
@@ -0,0 +1,47 @@
+---
+suite: test es credentials secret manifest
+templates:
+  - secret-elasticsearch.yaml
+tests:
+  - it: should have empty credentials as default
+    values: &testvalues
+      - ../../../tests/values/test_values.yaml
+    set:
+      global:
+        elasticsearch:
+          user: null
+          password: null
+    asserts:
+      - isEmpty:
+          path: data.ELASTICSEARCH_USERNAME
+      - isEmpty:
+          path: data.ELASTICSEARCH_PASSWORD
+  - it: should have credentials populated when global credentials are set
+    values: *testvalues
+    set:
+      global:
+        elasticsearch:
+          user: admin
+          password: letmein
+    asserts:
+      - isNotEmpty:
+          path: data.ELASTICSEARCH_USERNAME
+      - isNotEmpty:
+          path: data.ELASTICSEARCH_PASSWORD
```

Comment:
why not testing for the resulting b64 str?

Reply:
typical issue of doing the same thing in two different places in two different days 😀